react-frontend/src/components/StyleComponents/card.js

```diff
@@ -154,14 +154,18 @@ export const CommunityCardStyled = styled(Link).attrs({
   className: 'cardRound',
 })`
   background: #003366;
```

Comment:
Could you add:
```
border: 5px solid rgb(242, 242, 242); 
```
This seems to make the UI less jumpy when the yellow border appears on hover. Makes the border the same color as the page background, so it's invisible.